sdk/python/feast/cli.py

```diff
@@ -68,6 +72,11 @@ def cli(ctx: click.Context, chdir: str):
     """
     ctx.ensure_object(dict)
     ctx.obj["CHDIR"] = Path.cwd() if chdir is None else Path(chdir).absolute()
+    try:
+        if log is not None:
+            logging.basicConfig(level=getattr(logging, log.upper()))
```

Comment:
Can you check `feast/__init__.py`? We're calling some `basicConfig` there too which should be collapsed into this method.